docs/source/peers/peers.md

```diff
@@ -312,40 +360,67 @@ peer cannot, on its own, update the ledger --- updating requires the consent of
+
+In addition to the client application and gateway peers, orderer nodes play a
+critical role in transactions that update the ledger. Before committing transactions
+to the ledger, orderer nodes do just that --- they order all submitted transactions
+to ensure a consistent and accurate sequence of transactions on the shared ledger.
+Note that blockchain manages inconsistent ledgers through a process called forking,
+and the subsequent pruning (discarding and archiving) of the invalidated forks.
 
 ### Phase 1: Proposal
 
-Phase 1 of the transaction workflow involves an interaction between an
-application and a set of peers --- it does not involve orderers. Phase 1 is only
-concerned with an application asking different organizations' endorsing peers to
-agree to the results of the proposed chaincode invocation.
-
-To start phase 1, applications generate a transaction proposal which they send
-to each of the required set of peers for endorsement. Each of these *endorsing peers* then
-independently executes a chaincode using the transaction proposal to
-generate a transaction proposal response. It does not apply this update to the
-ledger, but rather simply signs it and returns it to the application. Once the
+Phase 1 of the transaction workflow involves an interaction between a client
+application and a set of peers --- orderers are not involved yet. In Phase 1,
+the client application initiates a request to endorsing peers, through the
+Fabric Gateway service, to return their responses to the proposed transaction.
+
+To begin phase 1, the client application generates a transaction proposal and
+sends it to a gateway peer in its own organization. The gateway peer with
+the highest block height is selected; if none are discovered, the gateway
+service moves on to select a gateway peer in another organization.
+
+Once selected, the gateway peer validates the transaction by invoking the
+chaincode without a ledger update --- this can be referred to as simulating
+the transaction, because it runs the application without any effect on the ledger
+or state on any peer. The gateway peer then returns it transaction result to the
+client, and if the application delegates endorsement to the gateway service,
+it also forwards the proposal to the *endorsing peers*.
+
+Each endorsing peer then follows suit, executing the chaincode called by the
+client application to generate a transaction proposal response and return it
+(signed) to the gateway peer. The gateway peer collects all responses, and if
+the responses collectively satisfy the endorsement policy (or policies), it
+forwards the transaction to the ordering service for sequencing. If the
+endorsement policies are not satisfied, or if the transaction would write to
+a private data collection, the gateway peer returns a failed proposal message
+to the client. The client then must explicitly specify the organizations
+required for endorsement and resubmit the updated transaction proposal.
```

Comment:
The client would not delegate to gateway service, and then upon failure, specify endorsing organizations.
These are two distinct application patterns, an application will EITHER delegate to gateway service OR specify endorsing organizations. It won't do some hybrid based on dynamic error handling.

Reply:
right the client application would have to specify the endorsing orgs and resubmit, it says the client must do this. are you saying the gateway service would have to be disabled on the peer first in order to resubmit specified orgs.?

Reply:
I'm saying the client would never delegate to gateway service and then upon failure resubmit to specific endorsing organizations. The choice of whether to delegate to gateway service or target specific endorsing organizations would be baked into the application based on application requirements. For example if you have private data or only trust a subset of organizations, then you would target those organizations. If you don't have these types of requirements, then you would delegate the choice to gateway service.

Reply:
no disabling of gateway. only v2.3 or v2.4 SDKs